package.json

```diff
@@ -6,7 +6,9 @@
   "scripts": {
     "start": "webpack-dev-server --mode development --open --hot",
     "build": "webpack --mode production",
-    "lint": "eslint src --fix"
+    "lint": "eslint src --fix",
+    "test": "jest",
+    "test:watch": "jest --watch"
```

Comment:
Add a script to collect coverage as well

Reply:
I have worked on this feedback.